gem/templates/malawi/base.html

```diff
@@ -18,10 +18,13 @@
         {% if self.image %}
           {% image self.image fill-450x200 as article_photo %}
         {% endif %}
+
         <meta property="og:title" content= "{% if self.social_media_title %}{{ self.social_media_title }}{% elif self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}" />
-        <meta property="og:description" content="{% if self.social_media_description %}{{ self.social_media_description }}{% else %}{{ self.subtitle }}{% endif %}"/>
+        <meta property="og:description" content="{% if self.search_description %}{{ self.search_description }}{% elif self.social_media_description %}{{ self.social_media_description }}{% else %}{{ self.subtitle }}{% endif %}"/>
+
         <meta property="og:url" content="{{ request.build_absolute_uri }}" />
-        <meta property="og:image" content="{% if self.social_media_image %}{{ tmp_photo.url }}{% elif article_photo.url %}{{ article_photo.url }}{% else %}{{request.site.root_url}}{% static 'img/zathu-fb-share.png' %}{% endif %}"/>
+        <meta property="og:image" content="{% if self.social_media_image %}{{ tmp_photo.url }}{% elif article_photo.url %}{{ article_photo.url }}{% else %}{{request.site.root_url}}{% static 'img/zathu.png' %}{% endif %}"/>
+
```

Comment:
shouldn't we use img/zathu-fb-share.png instead? its got a margin that would better display when shared